### PR TITLE
Impl math split

### PR DESCRIPTION
**Pull request by kriw**
It enables to split expression link `1 + 1 2 3 + 3` to `1 + 1`, `2`, `3`

### CODE REVIEW

**Comment by Maijin** [comment]
Regarding the output of travis, those are already fixed in master.

**Comment by kriw** [comment]
This is not for passing test, but for improving usability.
Before this patch, `1 + 1` is interpreted as `1`, `+`, `1` (spaces should be omitted).
And this patch enables to parse `1 + 1` as it is in `? ` and `?btw`

**Comment by radare** [comment]
imho if we want to split operations that should be with , or ;. the current behaviour sucks quite a lot because having so many output formats for a many operations is useless and noisy, and the space is a very common char to be used to allow a math op. i guess this pr fixed this regression. but didnt had a chance to test yet

File: libr/util/unum.c [hunk 1]
```diff
@@ -591,26 +577,85 @@ R_API ut64 r_num_tail(RNum *num, ut64 addr, const char *hex) {
...
+			len = i;
+			st = 1;
+			break;
+		case 1: //op
+			while (str[i] != '\0' && str[i] == ' ') {
+				i++;
 			}
+			if (!r_num_is_op (str[i])) {
+				return len;
+			} else if (str[i] == ')') {
```
Comment (radare) [inline libr/util/unum.c]: no need to else after return

File: libr/util/unum.c [hunk 2]
```diff
@@ -591,26 +577,85 @@ R_API ut64 r_num_tail(RNum *num, ut64 addr, const char *hex) {
...
+				return i+1;
```
Comment (radare) [inline libr/util/unum.c]: missing spaces around +

**Comment by radare** [comment]
Thanks! merged and fixed that stuff

**Pull request closed without merging**